### PR TITLE
multiswipe menu: keep the same parent page for going up from submenu

### PR DESCRIPTION
**Pull request by yparitcher**
~~Code borrowed from ReaderFooter~~

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/8474)

### CODE REVIEW

**Review by Frenzie — APPROVED** [review]

**Comment by Frenzie** [comment]
NB failed check (no time to look)

File: plugins/gestures.koplugin/main.lua [hunk 1]
```diff
@@ -297,6 +297,11 @@ function Gestures:genMultiswipeMenu()
     for _, item in ipairs(multiswipe_list) do
         table.insert(sub_item_table, self:genSubItem(item[1], item[2]))
     end
+    -- multiswipe menu: keep the same parent page for going up from submenu
+    for i = 1, #sub_item_table do
+        sub_item_table[i].menu_item_id = i
```
Comment (poire-z) [inline plugins/gestures.koplugin/main.lua:302]: May be you can avoid this additional loop, and just add in Gestures:genSubItem() line 24x: `menu_item_id = ges,` (if `ges` is a string, it should work with strings).

**Comment by yparitcher** [comment]
> NB failed check (no time to look)

https://github.com/koreader/koreader/pull/8473#issuecomment-976107901

File: plugins/gestures.koplugin/main.lua (hunk 1 above)
Comment (yparitcher) [reply plugins/gestures.koplugin/main.lua]: Thanks

**Review by poire-z — APPROVED** [review]

**Pull request merged**